include/seqan3/core/semiregular_box.hpp

```diff
@@ -0,0 +1,27 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides seqan3::semiregular_box.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <range/v3/utility/semiregular_box.hpp>
+
+#include <seqan3/core/platform.hpp>
+
+namespace seqan3
+{
+// Use semiregular box from ranges library.
+using ::ranges::semiregular_box;
+
+// Use semiregular box transformation trait from ranges library.
+using ::ranges::semiregular_box_t;
```

Comment:
doxygen may complain, see https://github.com/seqan/seqan3/blob/master/include/seqan3/core/common_tuple.hpp for example

Reply:
I could run the documentation without issues. Let's see what the CI says.

Reply:
Ok, I saw, that we also referenced it in the documentation so I decided to make give it a proper documentation entry.